docs/core/tools/global-json.md

```diff
@@ -113,6 +113,17 @@ The following example shows how to use the exact specified version:
 }
 ```
 
+The following example shows how to use the latest feature band and patch version installed of a specific major and minor version (in the form, 3.1.xxx):
```

Comment:
I don't think the clause in parentheses is necessary, but if you decide to keep it I would remove the comma.
```suggestion
The following example shows how to use the latest feature band and patch version installed of a specific major and minor version:
```

Reply:
Addressing this in another PR: https://github.com/dotnet/docs/pull/17990